torch/_inductor/lowering.py

```diff
@@ -5574,14 +5574,20 @@ def inner_fn(idx):
 
 
 @register_lowering(triton_kernel_wrapper_mutation)
-def triton_kernel_wrap_(*, kernel_idx, constant_args_idx, grid, kwargs):
+def triton_kernel_wrap_(
+    *, kernel_idx, constant_args_idx, grid, kwargs, tensors_to_clone=None
```

Comment:
Seems that you've updated the only call site of `triton_kernel_wrap_`. If so, let's make `tensors_to_clone` mandatory?

Reply:
This one matches the arguments of `triton_kernel_wrapper_mutation ` which does not need `tensors_to_clone`, so he does need that to be optional